### DISCUSSION TITLE
Does this work with any version combination?

### DISCUSSION THREAD

**Discussion by malkere** [volotat/SD-CN-Animation#196] [Q&A]
I'm trying to get txt2vid to work in any way I can... The first frame renders as normal and then it just devolves into a blur for every frame after that, producing a warning on each frame beyond the first:

*** Error running before_process: E:\stable-diffusion-webui-1.4.0\extensions-builtin\extra-options-section\scripts\extra_options_section.py
    Traceback (most recent call last):
      File "E:\stable-diffusion-webui-1.4.0\modules\scripts.py", line 466, in before_process
        script.before_process(p, *script_args)
      File "E:\stable-diffusion-webui-1.4.0\extensions-builtin\extra-options-section\scripts\extra_options_section.py", line 40, in before_process
        for name, value in zip(self.setting_names, args):
    TypeError: 'NoneType' object is not iterable

ControlNet - WARNING - No ControlNetUnit detected in args. It is very likely that you are having an extension conflict.Here are args received by ControlNet: [None, False, None]

v1.6 = same error
v1.5 = same error
v1.4 = same error

I'm just leaving the default settings as is and typing in "lion" into the prompt.

These are all fresh installs with only controlnet added since it's listed as a prerequisite. Are there any known workflows to get this working with new or older versions of webui?

**Comment by ricardofeynman** [comment]
https://github.com/volotat/SD-CN-Animation/issues/203#issuecomment-1904203379
